content/spin/configuration.md

```diff
@@ -168,12 +168,12 @@ variable provider and vault config provider.
 
 The environment variable provider which gets config values from the `spin` process's
 environment (_not_ the component `environment`). Config keys are translated
-to environment variables by upper-casing and prepending with `SPIN_APP_`:
+to environment variables by upper-casing and prepending with `SPIN_CONFIG_`:
```

Comment:
I'm afraid this has moved to `dynamic-configuration.md`.  The duplication here is (hopefully) temporary because this page is becoming more of a manifest guide (PR for this is in flight).  Apologies for the confusion and pardon our dust!

Reply:
Bleh, I just failed to pull before making the change.